Makefile

```diff
@@ -53,7 +58,15 @@ LDFLAGS='-extldflags "-static"'
 controller-bin:
 	@echo '********'
 	@echo 'Compiling controller binary'
-	docker run -it --rm -v $(REPO_ROOT):$(WORK_DIR):Z -w $(WORK_DIR) -e GOOS=$(GOOS) -e GOARCH=$(ARCH) -e CGO_ENABLED=$(CGO_ENABLED) $(BUILD_IMAGE) go build -a -ldflags $(LDFLAGS) -o $(WORK_DIR)/bin/$(CONTROLLER_BIN) $(WORK_DIR)/cmd/controller/controller.go
+	$(eval DOCKER_CMD_COUNT=$(shell docker ps | wc -c))
+	export DOCKER_CMD_COUNT
```

Comment:
L61 should have set this var in the env already

Reply:
not that I can access from within that little context of the makefile, the only way I could get DOCKER_CMD_COUNT was to export, it's strange, but basically if I don't export and then try to access that var, I get empty